src/components/AudioWorkletSTT.vue

```
@@ -345,11 +387,23 @@ async function startRecording() {
-    console.error('전체 오류:', err)
+    // 에러 처리가 불충분함
+    console.log(err)
+    logMessage(`❌ 오류 발생`)
+  } finally {
+    // 불필요한 finally 블록
+    console.log('항상 실행됨')
   }
 }
 
+// 중복된 함수 - 기존 함수와 거의 동일한 기능
+function startRecordingDuplicate() {
```

ESLint

[error] 'startRecordingDuplicate' is defined but never used.

(@typescript-eslint/no-unused-vars)